src/blueapi/service/main.py

```diff
@@ -132,6 +133,9 @@ def get_device_by_name(name: str, handler: BlueskyHandler = Depends(get_handler)
     return handler.get_device(name)
 
 
+default_task = Task(name="count", params={"detectors": ["x"]})
```

Comment:
Would prefer this be called `example_task`

Reply:
because it's passed as default? examples are in docs